app/models/phylo_tree.rb

```diff
@@ -125,6 +131,50 @@ def monitor_job(throttle = true)
     save
   end
 
+  def kickoff
+    return unless [STATUS_INITIALIZED, STATUS_FAILED].include?(status)
+    self.command_stdout, self.command_stderr, command_status = Open3.capture3(job_command)
+    if command_status.exitstatus.zero?
+      output = JSON.parse(command_stdout)
+      self.job_id = output['jobId']
+      self.status = STATUS_IN_PROGRESS
+    else
+      self.status = STATUS_FAILED
+      LogUtil.log_err_and_airbrake("Phylo tree failed to kick off for #{name} (#{id}).")
+    end
+    save
+  end
+
+  private
+
+  # We need to create a visualization object here to register the new phylo_tree
+  # as a generic visualization. Other types of visualizations are saved on-demand only.
+  # Because phylo_trees are created explictly, the user will expect them to persist.
+  # TODO: (gdingle): destroy visualization objects when phylo_tree is destroyed,
+  # and rename when renamed.
+  def create_visualization
+    Visualization.create(
+      user: user,
+      visualization_type: "phylo_tree",
+      data: { treeId: id },
+      name: name,
+      # Samples of a PhyloTree are all samples of the project at the time of creation.
```

Comment:
I thought there was a step in the phylo tree wizard for adding additional samples that could be from other projects?

Reply:
I didn't know about this. I'll check.

Reply:
I can confirm that @jshoe is correct.

Reply:
Thanks, I'll change the PR to use the selected samples for the phylo tree.